charts/veritable-cloudagent/values.yaml

```diff
@@ -100,7 +100,7 @@ backupBeforeStorageUpdate: false
 ## @param connectionImageUrl veritable-cloudagent connection image url
 connectionImageUrl: ""
 ## @param webhookUrl veritable-cloudagent web hook url
-webhookUrl: ""
+webhookUrl: [""]
```

Comment:
Its probably worth having a look at how we do arrays that need to be interpolated into either secrets or ENVARs elsewhere
https://github.com/digicatapult/helm-charts/blob/8c53c161913deeda86c996a6d14f9757f4c16032/charts/veritable-ui/values.yaml#L86-L87

https://github.com/digicatapult/helm-charts/blob/8c53c161913deeda86c996a6d14f9757f4c16032/charts/veritable-ui/templates/secret-session-cookie.yaml#L17